dockertest/environment.py

```diff
@@ -49,10 +46,10 @@ def get_selinux_context(path):
     :return: SELinux context as a string or None if not set
     """
     try:
-        # may include null-termination in string
-        return getxattr(path, 'security.selinux').replace('\000', '')
+        # First list item is null-terminated string length
+        return selinux.getfilecon(path)[1]
     except IOError, xcpt:
-        if xcpt.errno == errno.EOPNOTSUPP:
+        if xcpt.errno == errno.ENOENT:
```

Comment:
libselinux raises a different exception.  The bigger thing was the previous behavior of this function when it was embedded in the test-code.  I was trying to preserve that instead of messing with the test itself too much.  Though I suppose I could try fixing the test, it's just out of scope for what I was intending here.  I'll take a look though, maybe it'll only take 19 minutes or less 😀